### PR TITLE
APM Ingestion Controls: document the sampling service dimension

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do? What is the motivation?

This PR documents a new dimension that was added to the ingested_bytes APM usage metric: sampling service

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by rtrieu** [comment]
Created DOCS-8906 for editorial review.

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Ingested Bytes/s
 : Average number of bytes per second ingested into Datadog for the service over the past one hour.
 
 Downstream Bytes/s
-: Average number of bytes per second ingested for which the service *makes the sampling decision*. This includes the bytes of all downstream child spans that follow the decision made at the head of the trace, as well as spans caught by the [Error sampler][5], the [Rare sampler][6], and the [App Analytics][7] mechanism.
+: Average number of bytes per second ingested for which the service *makes the sampling decision*. This includes the bytes of all downstream child spans that follow the decision made at the head of the trace, as well as spans caught by the [Error sampler][5], the [Rare sampler][6], and the [App Analytics][7] mechanism. This column's data is based on the `sampling_service` dimension, set on the `datadog.estimated_usage.apm.ingested_bytes` metrics: for more information, read about [APM usage metrics][15].
```
Comment (brett0000FF) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:73]:
```suggestion
: Average number of bytes per second ingested for which the service *makes the sampling decision*. This includes the bytes of all downstream child spans that follow the decision made at the head of the trace, as well as spans caught by the [Error sampler][5], the [Rare sampler][6], and the [App Analytics][7] mechanism. This column's data is based on the `sampling_service` dimension, set on the `datadog.estimated_usage.apm.ingested_bytes` metrics. For more information, read [APM usage metrics][15].
```

File: content/en/tracing/trace_pipeline/metrics.md [hunk 2]
```diff
@@ -43,12 +43,20 @@ The following metrics are associated with ingested spans usage:
  - `datadog.estimated_usage.apm.ingested_spans`
  - `datadog.estimated_usage.apm.ingested_traces`
 
-Control the usage with `datadog.estimated_usage.apm.ingested_bytes`. Ingestion is metered as a volume, not as a number of spans or traces. This metric is tagged by `env` and `service` so you can spot which environments and services are contributing to the ingestion volume.
+Control the usage with `datadog.estimated_usage.apm.ingested_bytes`. Ingestion is metered as a volume, not as a number of spans or traces. This metric is tagged by `env`, `service` and `sampling_service` so you can spot which environments and services are contributing to the ingestion volume. Read more information about the `sampling_service` dimension [here](#what-is-the-sampling-service).
```
Comment (brett0000FF) [inline content/en/tracing/trace_pipeline/metrics.md:46]:
```suggestion
To control usage, use `datadog.estimated_usage.apm.ingested_bytes`. Ingestion is metered by volume, not by the number of spans or traces. This metric is tagged with `env`, `service`, and`sampling_service`. These tags help identify which environments and services contribute to the ingestion volume. For more information about the `sampling_service` dimension, read [What is the sampling service?](#what-is-the-sampling-service).
```

File: content/en/tracing/trace_pipeline/metrics.md [hunk 3]
```diff
@@ -43,12 +43,20 @@ The following metrics are associated with ingested spans usage:
...
 
 This metric is also tagged by `ingestion_reason`, reflecting which [ingestion mechanisms][5] are responsible for sending spans to Datadog. These mechanisms are nested in the tracing libraries of the Datadog Agent. For more information about this dimension, see the [Ingestion Reasons dashboard][6].
 
 The `datadog.estimated_usage.apm.ingested_traces` metric measures the number of requests sampled per second, and only counts traces sampled by [head-based sampling][7]. This metric is also tagged by `env` and `service` so you can spot which services are starting the most traces.
 
+#### What is the sampling service?
+
+The `sampling_service` dimension on the `datadog.estimated_usage.apm.ingested_bytes` assigns ingested bytes to the **service that makes the sampling decision**, not the service from which the span is emitted.
```
Comment (brett0000FF) [inline content/en/tracing/trace_pipeline/metrics.md:54]:
```suggestion
The `sampling_service` dimension on the `datadog.estimated_usage.apm.ingested_bytes` assigns ingested bytes to the **service that makes the sampling decision**, not the service emitting the span.
```

File: content/en/tracing/trace_pipeline/metrics.md [hunk 4]
```diff
@@ -43,12 +43,20 @@ The following metrics are associated with ingested spans usage:
...
 
 The `datadog.estimated_usage.apm.ingested_traces` metric measures the number of requests sampled per second, and only counts traces sampled by [head-based sampling][7]. This metric is also tagged by `env` and `service` so you can spot which services are starting the most traces.
 
+#### What is the sampling service?
+
+The `sampling_service` dimension on the `datadog.estimated_usage.apm.ingested_bytes` assigns ingested bytes to the **service that makes the sampling decision**, not the service from which the span is emitted. 
+
+Group the metric by `sampling_service` to spot which services are contributing the most to the total ingested volumes. For instance, if a service `A` starts the trace and a head-based sampling decision is made, then calls service `B` and `C`, all bytes from service `A`, `B`, and `C` will be attributed to `sampling_service:A` as the decision of sampling the trace was made on the service `A`.
```
Comment (brett0000FF) [inline content/en/tracing/trace_pipeline/metrics.md:56]:
```suggestion
Group the metric by `sampling_service` to identify services contributing the most to total ingested volumes. For instance, if service `A` starts a trace and makes a head-based sampling decision before calling services `B` and `C`, all bytes from services `A`, `B`, and `C` are attributed to `sampling_service:A`.
```

**Review by brett0000FF — APPROVED** [review]
@dussault-antoine - Thanks! This looks good. I just left a few minor suggestions.

**Comment by dussault-antoine** [comment]
@brett0000FF good to merge ! thanks for the review

**Pull request merged**